src/main/java/com/test/KafkaTestApp/client/ClientManager.java

```diff
@@ -0,0 +1,24 @@
+package com.test.KafkaTestApp.client;
+
+import com.test.KafkaTestApp.config.KafkaConfig;
+import lombok.AllArgsConstructor;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class ClientManager {
+
+  private final KafkaTemplate<String, Client> kafkaTemplate;
+  private final KafkaConfig kafkaConfig;
+
+  public void create(Client client) {
+    kafkaTemplate.send(kafkaConfig.getClientTopic(), client);
```

Comment:
may you produce Client into Kafka in JSON format (feel free to use Jackson ObjectMapper)?